server/src/test/java/org/opensearch/index/translog/LocalTranslogTests.java

```diff
@@ -519,18 +519,22 @@ public void testStats() throws IOException {
                 builder.startObject();
                 copy.toXContent(builder, ToXContent.EMPTY_PARAMS);
                 builder.endObject();
-                assertThat(
-                    builder.toString(),
-                    equalTo(
-                        "{\"translog\":{\"operations\":4,\"size_in_bytes\":"
-                            + 326
-                            + ",\"uncommitted_operations\":4,\"uncommitted_size_in_bytes\":"
-                            + 271
-                            + ",\"earliest_last_modified_age\":"
-                            + stats.getEarliestLastModifiedAge()
-                            + "}}"
-                    )
-                );
+                String expectedLocalTranslogStats = "{\"translog\":{\"operations\":4,\"size_in_bytes\":"
+                    + 326
+                    + ",\"uncommitted_operations\":4,\"uncommitted_size_in_bytes\":"
+                    + 271
+                    + ",\"earliest_last_modified_age\":"
+                    + stats.getEarliestLastModifiedAge();
+                String defaultRemoteStoreStats = ",\"remote_store\":{\"upload\":{"
+                    + "\"total_uploads\":{\"started\":0,\"failed\":0,\"succeeded\":0},"
+                    + "\"total_uploads_in_bytes\":{\"started\":0,\"failed\":0,\"succeeded\":0}"
+                    + "}}";
+                // TODO: if (out.getVersion().onOrAfter(Version.V_2_10_0)) {
+                if (out.getVersion().onOrAfter(Version.CURRENT)) {
+                    assertEquals(expectedLocalTranslogStats + defaultRemoteStoreStats + "}}", builder.toString());
+                } else {
+                    assertEquals(expectedLocalTranslogStats + "}}", builder.toString());
```

Comment:
are both the conditions tested for this test through some randomness? Does not look so.

Reply:
Ack, will update this

Reply:
Based on the discussion over call, removed the version check to always assert with remote store stats (0 values in this case). Resolving this thread. Please reopen if needed.